server/routes/book/postBook.js

```diff
@@ -37,7 +39,7 @@ const upload = multer({
 });
 
 router.post(
-  '/api/books',
+  '/api/books', auth, 
```

Comment:
narazie to to mało sensu ma bo nie sprawdzamy czy ksiązka należy do ownera, ale to i tak finalnie musi być więc może zostać